functions/index.js

```diff
@@ -78,3 +90,33 @@ exports.updateBlogViews = functions.https.onRequest((request, response) => {
       .catch(error => response.status(500).send(error));
   });
 });
+
+exports.httpEmail = functions.https.onRequest((req, res) => {
+  cors(req, res, () => {
+    verifyOrigin(req, res);
+
+    if (req.method !== 'POST') {
+      const error = new Error('Only POST requests are accepted');
+      res.status(405).send(error)
+    }
+
+    const request = sendgridClient.emptyRequest({
+      method: 'POST',
+      path: '/v3/mail/send',
+      body: configureEmailBody(req.body)
+    });
+
+    sendgridClient.API(request)
+      .then((response) => {
+        if (response.body) {
+          return res.status(200).send(response.body);
+        } else {
+          return res.end();
+        }
```

Comment:
@dmeneses shouldn't this be like:
```
if (...) {
  return ...;
}
return ...;
```
your else doesn't have any additional condition.

Reply:
Fixed, thanks!